### PR TITLE
Cypress root & child command

### PR DESCRIPTION
**Pull request by animehart**
## Summary
Added:
1. Root escalation checks for Sudo su
2. Commands in Child process on session view structure check (check for correct number of commands after user expands child command expand button and make sure child commands is correctly placed)

### CODE REVIEW

**Comment by mitodrummer** [comment]
Forgive my ignorance, as I have not been super involved the last working group, but is there a reason the cypress tests are apart of the security_solution plugin and not session_view plugin?

File: x-pack/plugins/security_solution/cypress/integration/hosts/session_view.spec.ts [hunk 1]
```diff
@@ -132,5 +134,47 @@ describe('Session view', () => {
...
+          expect(parentCommandLeftMargin).to.be.greaterThan(childCommandLeftMargin)
+        })
+      })
+    })
+    
+    it('root escalation', () => {
+      openSessionView(TEST_EVENT_ID);
+
+      // Get the parent div for Root Escalation button and checks if Sudo su and Root Escalation button is with same parent or not
+      cy.contains('Root escalation').parent().contains('sudo').should('exist');
```
Comment (zizhouW) [inline x-pack/plugins/security_solution/cypress/integration/hosts/session_view.spec.ts:175]: Right now it only checks the first root escalation. Might want to check if every root escalation corresponds to a `sudo`

File: x-pack/plugins/security_solution/cypress/integration/hosts/session_view.spec.ts [hunk 2]
```diff
@@ -132,5 +134,47 @@ describe('Session view', () => {
...
+      openSessionView(TEST_EVENT_ID);
+
+      // Amount of visible commands on the session view should increase when user clicks on the Child Process dropdown button
+      cy.get(SESSION_COMMANDS).children().its('length').then(lengthBefore =>{
+        const beforeClick = lengthBefore;
+        cy.contains('Child processes').click()
+        cy.contains(FIRST_CHILD_COMMAND).should('exist');
+        cy.get(SESSION_COMMANDS).children().its('length').then(lengthAfter =>{
+          const afterClick = lengthAfter;
+          expect(afterClick).to.be.greaterThan(beforeClick)
```
Comment (zizhouW) [inline x-pack/plugins/security_solution/cypress/integration/hosts/session_view.spec.ts:148]: We can just use `lengthAfter` directly here, without declaring `afterClick`

**Comment by zizhouW** [comment]
@mitodrummer we moved the cypress tests to security solutions plugin because the session leader table and the session view are integrated to under `app/security/hosts/sessions`. I think it's more organized to keep it there as a part of their `host` cypress tests since i dont think we will be keeping the session view page. We will also need to remove the cypress stuff in session_view though.

**Review by zizhouW — APPROVED** [review]

**Pull request merged**